website/src/components/index-page/index.tsx

```diff
@@ -0,0 +1,33 @@
+import { Hero } from "./hero"
+import { PredictableResults } from "./predictable-results"
+import { SingleRequest } from "./single-request"
+import { TypeSystem } from "./type-system"
+import { PowerFulTools } from "./powerful-tools"
+import { WithoutVersion } from "./without-version"
+import { BringYourOwnCode } from "./bring-your-own-code"
+import { WhoIsUsing } from "./who-is-using"
+
+export function IndexPage() {
```

Comment:
<img width="1727" alt="Screenshot 2024-03-19 at 10 35 42" src="https://github.com/graphql/graphql.github.io/assets/35999252/1201af60-fc1c-41e2-94ad-751044ec3afc">

Something jarring about each slide here, it looks like there is more space/padding on the right to the left - shifting the view to the left.